### PR TITLE
add schema.js, initialize npm package

### PR DESCRIPTION
**Pull request by lauripiisang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+const validatePostHelloCouchDb = new Schema({
+    dataSchema:{
+        _id: Joi.string().required(),
+        _rev: Joi.string().required(),
+        hello: Joi.string().allow('').required(),
+        goodbye: Joi.string().allow('').required(),
+    },
+    createIgnoreKeys: ['_id', '_rev'],
+    updateIgnoreKeys: ['_id'],
+    xor: ['hello', 'goodbye']
```
Comment (KaskMartin) [inline README.md]: add bulkUpdateIgnoreKeys property to the example.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+});
+```
```
Comment (KaskMartin) [inline README.md]: Maybe add an examples of Schema usage also. Ex.:
```
const validatedData = await validatePostHelloCouchDb.getCreateData({
        _id: '123',
        _rev: '321',
        hello: 'Hello World',
        nonsense: 'Foo',
});
// validatedData = {hello =  'Hello World'}
```

File: README.md (hunk 1 above)
Comment (KaskMartin) [inline README.md]: Maybe explain the properties, ex:
```
createIgnoreKeys: ['_id', '_rev'],  // keys to be stripped on getCreateData() call
updateIgnoreKeys: ['_id'], // keys to be stripped on getUpdateData() call
bulkUpdateIgnoreKeys:  ['_id'], // keys to be stripped on getBulkUpdateData() call
xor: ['hello', 'goodbye'] // validation checks that one and only one of the xor keys is present
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "keywords": [
+    "joi",
+    "schema",
+    "database",
+    "update",
+    "create",
+    "api"
+  ],
+  "author": "Toomas Sõukand <toomas.soukand@pipedrive.com>",
```
Comment (jaanuspoldre) [inline package.json]: dont give this guy all the glory

Comment (lauripiisang) [reply package.json]: None of the glory, all of the blame

**Pull request merged**